charts/pega/charts/hazelcast/values.yaml

```diff
@@ -1,11 +1,25 @@
 ---
 image: "YOUR_HAZELCAST_IMAGE:TAG"
+clusteringServiceImage: "YOUR_CLUSTERING_SERVICE_IMAGE:TAG"
 imagePullPolicy: "Always"
 # Enter the number of initial members in Hazelcast cluster.
 replicas: 3
 # Setting below to true will deploy the infinity in client-server Hazelcast model.
 # Note: Make sure to set this value as "false" in case of platform version below "8.6". If not set this will fail the installation.
 enabled: true
+clusteringServiceEnabled: false
+
+# Setting related to Hazelcast migration.
+migration:
+  # Setting up to initiate the migration job.(Only for CMC/CloudK have InfinityOps service for migration task.)
+  enabled: false
+  # Migration job docker image with bash and kubernetes
+  migrationJobImage: "YOUR_MIGRATION_JOB_IMAGE:TAG"
+  # Setting up to true while starting the new hazelcast pods in case of embedded to c/s migration process.
+  embeddedToCSMigration: false
+  # Setting up to true while deleting the old hazelcast pods after Migration.
```

Comment:
Its not clear what is `skipRestart` flag for? Could not correlate with the the description and param name. Is this something open for clients to configure?

Reply:
@kishorv10 , updated as below:
Setting up to true during a deployment that removes an older Hazelcast cluster to avoid restarting of Pega pods, which can cause the migration to fail.